internal/locale/locale.go

```diff
@@ -155,8 +156,13 @@ func Tt(translationID string, args ...interface{}) string {
 
 	// For templates we want to manually specify the linebreaks as the way YAML gets parsed makes
 	// this very painful otherwise
-	translation = strings.Replace(translation, "\n", "", -1)
+
+	// Replace newlines in yaml strings with space to avoid concatenated words
+	replaceRegex := regexp.MustCompile(`[\ ]?\n(\w|\{)`)
+	translation = replaceRegex.ReplaceAllString(translation, " $1")
 	translation = strings.Replace(translation, "{{BR}}", "\n", -1)
+	// Avoid indentation after newlines
+	translation = strings.Replace(translation, "\n ", "\n", -1)
```

Comment:
Can drop this if you use my suggestion

Reply:
This is still required to avoid a space on the newlines we purposely added to the yaml strings